src/Datasource/EntityTrait.php

```diff
@@ -34,22 +34,22 @@ trait EntityTrait
      *
      * @var array
      */
-    protected $_fields = [];
+    protected array $_fields = [];
 
     /**
      * Holds all fields that have been changed and their original values for this entity.
      *
      * @var array
      */
-    protected $_original = [];
+    protected array $_original = [];
 
     /**
      * List of field names that should **not** be included in JSON or Array
      * representations of this Entity.
      *
      * @var array<string>
      */
-    protected $_hidden = [];
+    protected array $_hidden = [];
```

Comment:
These are going to be painful for users and we'll need to have rector support for this. I've added these changes to the wiki list.